### PR TITLE
Docathon][Add CN Doc No.56-57]

### PR DESCRIPTION
**Pull request by zade23**
### PR types

Others

### PR changes

Docs 

#### 中文文档添加任务 
#6193 

#### 新增中文文档：

- fused_bias_dropout_residual_layer_norm  
- FusedBiasDropoutResidualLayerNorm

#### 英文文档链接：

| 序号 | API名称                                                      | 英文文档地址                                                 |
| ---- | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 56   | paddle.incubate.nn.functional.fused_bias_dropout_residual_layer_norm | https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_en.html |
| 57   | paddle.incubate.nn.FusedBiasDropoutResidualLayerNorm         | https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_en.html |

@ZzSean @sunzhongkai588

### CODE REVIEW

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. _cn_api_paddle_incubate_nn_functional_fused_bias_dropout_residual_layer_norm:
+
+fused_bias_dropout_residual_layer_norm
+-------------------------------
+
+.. py:function:: paddle.incubate.nn.functional.fused_bias_dropout_residual_layer_norm(x, residual, bias=None, ln_scale=None, ln_bias=None, dropout_rate=0.5, ln_epsilon=1e-05, training=True, mode='upscale_in_train', name=None)
+
+融合偏置、Dropout 和残差层归一化操作符。其伪代码如下：
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: ”融合偏置、Dropout 和残差层归一化“不需要翻译，直接饮用原操作符名。如改为“fused_bias_dropout_residual_layer_norm操作符。”

若想增加功能介绍，可修为形式 ”fused_bias_dropout_residual_layer_norm操作符， 包含融合偏置、Dropout 和残差层归一化。“

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+::::::::::::
+    - **x** (Tensor) - 输入张量。其形状为 `[*, embed_dim]`。
+    - **residual** (Tensor) - 残差张量。其形状与 x 相同。
+    - **bias** (Tensor，可选) - 线性的偏置。其形状为 `[embed_dim]`。默认为 None。
+    - **ln_scale** (Tensor，可选) - 层归一化的权重张量。其形状为 `[embed_dim]`。默认为 None。
+    - **ln_bias** (Tensor，可选) - 层归一化的偏置张量。其形状为 `[embed_dim]`。默认为 None。
+    - **dropout_rate** (float，可选) - 在注意力权重上使用的 Dropout 概率，用于在注意力后的 Dropout 过程中丢弃一些注意力目标。0 表示无 Dropout。默认为 0.5。
+    - **ln_epsilon** (float，可选) - 在层归一化的分母中添加的小浮点数，用于避免除以零。默认为 1e-5。
+    - **training** (bool，可选) - 表示是否处于训练阶段的标志。默认为 True。
+    - **mode** (str，可选) - ['upscale_in_train'(默认) | 'downscale_in_infer']，有两种模式：
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: “有两种模式” -> "两种模式分别为："

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.. _cn_api_paddle_incubate_nn_FusedBiasDropoutResidualLayerNorm:
+
+FusedBiasDropoutResidualLayerNorm
+-------------------------------
+
+.. py:class:: paddle.incubate.nn.FusedBiasDropoutResidualLayerNorm(embed_dim, dropout_rate=0.5, weight_attr=None, bias_attr=None, epsilon=1e-05, name=None)
+
+应用融合偏置、Dropout 和残差层归一化操作。
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: 使用原算子名，无需翻译，如果要解释操作，请在后续补充。
推荐修改为
”应用fused_bias_dropout_residual_layer_norm操作符，包含融合偏置、Dropout 和残差层归一化操作。“

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+                                 1. upscale_in_train(默认)，在训练时上调输出
+                                    - 训练：out = input * mask / (1.0 - p)
+                                    - 推理：out = input
+
+                                 2. downscale_in_infer，在推理时下调输出
+                                    - 训练：out = input * mask
+                                    - 推理：out = input * (1.0 - p)
+    - **name** (str，可选) - 操作的名称（可选，默认为 None）。更多信息，请参考：ref:`api_guide_Name`。
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: 此处的“：ref:`api_guide_Name`” 同FusedBiasDropoutResidualLayerNorm（上一处填充）中文文档中的“: ref:`cn_api_paddle_ParamAttr`” 不一致，请统一格式

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    - **dropout_rate** (float，可选) - 在注意力权重上使用的 Dropout 概率，用于在注意力后的 Dropout 过程中丢弃一些注意力目标。0 表示无 Dropout。默认为 0.5。
+    - **bias_attr** (ParamAttr|bool，可选) - 指定偏置参数的属性。默认为 None，意味着使用默认的偏置参数属性。如果设置为 False，则该层不会有可训练的偏置参数。具体用法请参见 :ref:`cn_api_paddle_ParamAttr` 。
+    - **epsilon** (float，可选) - 添加到方差中的小值，以防止除零。默认为 1e-05。
+
+代码示例
+::::::::::::
+
+COPY-FROM: paddle.incubate.nn.FusedBiasDropoutResidualLayerNorm
+
+forward(src, src_mask=None, cache=None)
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: 此处和[英文文档](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_en.html)无法对应，请再次确定来源是否有误

应修改为forward(x, residual)，代码中也是此格式，请本人也重新确认一下。

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+::::::::::::
+应用融合偏置、Dropout 和残差层归一化操作。
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: 使用原算子名，无需翻译，如果要解释操作，请在后续补充。
推荐修改为
”应用fused_bias_dropout_residual_layer_norm操作符，包含融合偏置、Dropout 和残差层归一化操作。“

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+参数
+::::::::::::
+    - **x** (Tensor) - 输入张量。它是一个形状为 `[batch_size, seq_len, embed_dim]` 的张量。数据类型应为 float32 或 float64。
+    - **residual** (Tensor，可选) - 残差张量。它是一个形状为 `[batch_size, value_length, vdim]` 的张量。数据类型应为 float32 或 float64。
+
+返回
+::::::::::::
+Tensor|tuple：数据类型与 ``x`` 一样。
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: 根据[英文文档](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_en.html) 应翻译为 “与x具有相同数据类型和形状的张量”

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+extra_repr()
+::::::::::::
+返回当前层的额外信息。
```
Comment (Vvsmile) [inline docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: 根据[英文文档](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_en.html) 应翻译为“当前层的额外表示，可以有使用者建立的层的自定义实现”（可以想一下更好的翻译）

**Review by Vvsmile — COMMENTED** [review]
请修改Review处，非常感谢您的贡献！

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst (hunk 1 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: Done
修改前：`融合偏置、Dropout 和残差层归一化操作符。` 
修改后：`fused_bias_dropout_residual_layer_norm 操作符，包含融合偏置、Dropout 和残差层归一化。 `

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst (hunk 2 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: Done

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst (hunk 3 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: Done
修改前：`应用融合偏置、Dropout 和残差层归一化操作。` 
修改后：`应用fused_bias_dropout_residual_layer_norm操作符，包含融合偏置、Dropout 和残差层归一化操作。 `

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst (hunk 4 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: @Vvsmile 开发者您好
关于这两处引用的格式，可能是正确的。
我在中文文档中找到了同时引用(`Name` 和 `ParamAttr`)的文档[FusedMultiHeadAttention](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/incubate/nn/FusedMultiHeadAttention_cn.html#fusedmultiheadattention)。
image

若是我理解错您的意思了，或有其他方面的修改需求，烦请更新Review

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst (hunk 5 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: Done
已确认，错误表述可能是翻译时函数对应错误。
修改为：`forward(x, residual)`

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst (hunk 6 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: Done
修改前：`应用融合偏置、Dropout 和残差层归一化操作。` 
修改后：`应用fused_bias_dropout_residual_layer_norm操作符，包含融合偏置、Dropout 和残差层归一化操作。 `

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst (hunk 7 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: Done

File: docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst (hunk 8 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/FusedBiasDropoutResidualLayerNorm_cn.rst]: Done
修改前：`返回当前层的额外信息。`
修改为：`当前层的额外表示，您可以自定义实现自己的层。`

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst (hunk 4 above)
Comment (Vvsmile) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: 细节问题，统一一下ref前是中文冒号还是英文冒号，表述不清引起的额外工作量，烦请见谅

Comment (zade23) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: > 细节问题，统一一下ref前是中文冒号还是英文冒号

了解了，目前修改中已经统一使用英文冒号了

Comment (Vvsmile) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: 辛苦

**Comment by Vvsmile** [comment]
LGTM

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+    - **ln_scale** (Tensor，可选) - 层归一化的权重张量。其形状为 `[embed_dim]`。默认为 None。
+    - **ln_bias** (Tensor，可选) - 层归一化的偏置张量。其形状为 `[embed_dim]`。默认为 None。
+    - **dropout_rate** (float，可选) - 在注意力权重上使用的 Dropout 概率，用于在注意力后的 Dropout 过程中丢弃一些注意力目标。0 表示无 Dropout。默认为 0.5。
+    - **ln_epsilon** (float，可选) - 在层归一化的分母中添加的小浮点数，用于避免除以零。默认为 1e-5。
+    - **training** (bool，可选) - 表示是否处于训练阶段的标志。默认为 True。
+    - **mode** (str，可选) - ['upscale_in_train'(默认) | 'downscale_in_infer']，两种模式分别为：
+
+                                 1. upscale_in_train(默认)，在训练时上调输出
+                                    - 训练：out = input * mask / (1.0 - p)
+                                    - 推理：out = input
```
Comment (sunzhongkai588) [inline docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]:
```suggestion
                                 1. upscale_in_train(默认)，在训练时上调输出
                                 
                                    - 训练：out = input * mask / (1.0 - p)
                                    - 推理：out = input
```
image
加个空行吧

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+                                 2. downscale_in_infer，在推理时下调输出
+                                    - 训练：out = input * mask
+                                    - 推理：out = input * (1.0 - p)
```
Comment (sunzhongkai588) [inline docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]:
```suggestion
                                 2. downscale_in_infer，在推理时下调输出
                                 
                                    - 训练：out = input * mask
                                    - 推理：out = input * (1.0 - p)
```

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst (hunk 9 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: Done

File: docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst (hunk 10 above)
Comment (zade23) [reply docs/api/paddle/incubate/nn/functional/fused_bias_dropout_residual_layer_norm_cn.rst]: Done

**Review by sunzhongkai588 — COMMENTED** [review]
好像得用 pre-commit 再跑一下

**Comment by zade23** [comment]
> 好像得用 pre-commit 再跑一下

Done
image

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
